src/lang/tk.ts

```diff
@@ -0,0 +1,9 @@
+const TK = ['sekunt', 'minut', 'sagat', 'gün', 'hepde', 'aý', 'ýyl'];
+function getSuffix(unit: string): string {
+  return unit.match(/[aouy]/) ? 'dan' : 'den';
```

Comment:
what is the meaning?

Reply:
`TK` is units (second, minute...) in turkmen. 
`getSuffix` function is for translating future dates. 

Please check this out: https://translate.google.com/#view=home&op=translate&sl=auto&tl=tk&text=in%202%20minutes